.yamllint

```diff
@@ -8,3 +8,4 @@ rules:
 ignore: |
   site/node_modules/*
   site/vendor/*
+  .github/workflows/cypress_ci_testing.yml
```

Comment:
This probably shouldn't be checked in.

Reply:
I thought it was a file created by cypress, but then I realized it was a biproduct of my testing on a forked repo, this has been removed.